### ISSUE TITLE
JSON.parse of readBreakpoints() errors with libsass 

### ISSUE THREAD

**Issue by herrkessler** [eduardoboucas/include-media-export#3]
Due to "variable interpolation causes additional string escaping" as in https://github.com/sass/libsass/issues/1115 the data object in readBreakpoints() returns an invalid string.

![JSON parse Error](http://herrkessler.de/github/JSONparseError.png)

and check out the CSS:

![CSS Output](http://herrkessler.de/github/LibsassEscapingErrors.png)

One could fix the RegExp of the [removeQuotes() function on line 80](https://github.com/eduardoboucas/include-media-export/blob/master/includeMedia.js#L80) to:

``` javascript
string = string.replace(/[']/g,'"').replace(/\\|^[\s\S]{0,1}|[\s\S]$/g, "");
```

This certainly not the best RegExp but works for me with libsass.

**Comment by eduardoboucas** [comment]
That's removing the curly braces though. How are you dealing with that?

**Comment by herrkessler** [comment]
Is the RegExp removing the curly braces for you at the front and the and of the data string? For me the new string looks like this:

![New String](http://herrkessler.de/github/JSONStringConsole.png)

``` javascript
{"phone":{"value": "320px", "active": true}, "tablet":{"value": "768px", "active": true}, "desktop":{"value": "1024px", "active": true}}
```

And everything works as it should. Do you use ruby sass?

**Comment by eduardoboucas** [comment]
I've used [SassMeister](http://sassmeister.com/gist/0f33426463aefc41ab61) to generate the JSON object with LibSass and then your function to parse it. Here's [a fiddle](http://jsfiddle.net/uv0o180d/).

**Comment by eduardoboucas** [comment]
Sorry, I was being an idiot. [This fiddle](http://jsfiddle.net/3dngLe76/) shows that it works.

Do you want to create a PR?

**Comment by eduardoboucas** [comment]
Fixed in #4.
